config/routes.rb

```diff
@@ -15,4 +16,8 @@
   end
     # devise_for :users, controllers: { confirmations: 'users/confirmations' }
   root to: 'users#index'
+  get 'api/user/:id/posts' => 'users#api_user_post', format: 'json'
+  get 'api/post/:id/comments' => 'posts#post_comments_api', format: 'json'  
+  post 'api/post/:id/comments' => 'comments#post_create_api'
```

Comment:
- [ ] I was unable to add a comment for a post via the API. I got back `500 internal server error`
Please spare some time to take a look at it. I will be glad to know if it works well on your end.

Error 1 Screenshot |
:-----------------------:|
<img width="932" alt="Screen Shot 2022-11-11 at 8 22 55 PM" src="https://user-images.githubusercontent.com/39523312/201426700-268ff58c-a44e-4b40-90eb-49e867938b03.png"> |



Error 2 Screenshot |
:--------------------------:|
<img width="935" alt="Screen Shot 2022-11-11 at 8 23 09 PM" src="https://user-images.githubusercontent.com/39523312/201426755-1a8dfc06-e2a3-422f-b041-46bbf8c83377.png"> |